app/ui/hooks.tsx

```diff
@@ -0,0 +1,2 @@
+export const useMagicLink = (s: string) => ({})
```

Comment:
From here on down are hacks that can be ignored that were added to try to get it working on the demo